.gitignore

```diff
@@ -13,6 +13,7 @@ composer.lock
 
 # Ignore local PHPUnit configuration
 /phpunit.xml
+/.phpunit.result.cache
```

Comment:
I'm assuming the `.phpunit.result.cache` helps unit tests run faster in subsequent runs. It might be a good idea to add it to `.travis.yml` as a cache. However, I don't think it is required in this repo since there are fewer tests.

Reply:
Result cache saves the previous test results for things like test-on-save: it shouldn't be cached when we want to run all the tests after every build.